Dockerfile

```diff
@@ -10,12 +10,24 @@ WORKDIR /usr/src
 ## Setup Home Assistant Core dependencies
 COPY requirements.txt homeassistant/
 COPY homeassistant/package_constraints.txt homeassistant/homeassistant/
+# hadolint ignore=DL4006
+RUN \ 
+   grep -v '^[#-]' < homeassistant/requirements.txt | xargs -P 6 -n 4 -- \
+   pip3 download --no-index --only-binary=:all: \
+   --find-links "${WHEELS_LINKS}" --use-deprecated=legacy-resolver \
+   --cache-dir "/tmp/pip_cache"
 RUN \
-    pip3 install --no-cache-dir --no-index --only-binary=:all: --find-links "${WHEELS_LINKS}" \
-    -r homeassistant/requirements.txt --use-deprecated=legacy-resolver
+    pip3 install --cache-dir "/tmp/pip_cache" --no-index --only-binary=:all: --find-links "${WHEELS_LINKS}" \
+    -r homeassistant/requirements.txt --use-deprecated=legacy-resolver ; exit 0
```

Comment:
Why do we need `exit 0` ? Doesn't that mean we never fail if installation fails ?

Reply:
Its on the wrong line 🤦

Reply:
Fixed in 0f0be7ee37297bffb52ea48ca8c35ca8ccf7cc55